src/PanelContent.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import HCAlogo from './assets/img/HCA-logo.jpg'
```

Comment:
This image is heavily compressed and looks quite muddy. You can just link to `/resources/images/logos/human_cell_atlas.png`.